lib/index.js

```diff
@@ -121,12 +121,13 @@ module.exports = {
 				});
 			},
 			unbind: function(rcpt,handler) {
-				var tmp = [];
+				var tmp = {};
```

Comment:
This was the main issue, which caused `handlers` to be an empty array after calling `unbind` once